pkg/actions/nodegroup/testdata/br-updated-template.json

```diff
@@ -157,8 +158,7 @@
         "Tags": {
           "alpha.eksctl.io/nodegroup-name": "br",
           "alpha.eksctl.io/nodegroup-type": "managed"
-        },
-        "Version": "1.27"
```

Comment:
Was this necessary? If yes, i'm curious to hear why

Reply:
On nodegroup upgrades, there is a CFN limitation in which if you specify `releaseVersion` OR `kubernetesVersion` you can't update any other property, also you can't update both at the same time. Therefore, even if we run something like `eksctl upgrade nodegroup --kubernetes-version 1.27`, we'll still try to fetch the latest release version, and if one is found, we won't pass kubernetes version to CFN template.

This was previously required when upgrading kubernetes version as we weren't fetching release version for Bottlerocket.